metagraph/src/graph/representation/bitmap/dbg_bitmap.cpp

```diff
@@ -37,7 +37,7 @@ DBGBitmap::DBGBitmap(DBGBitmapConstructor *builder) : DBGBitmap(2) {
 void DBGBitmap::map_to_nodes(std::string_view sequence,
                              const std::function<void(node_index)> &callback,
                              const std::function<bool()> &terminate) const {
-    for (const auto &[kmer, is_valid] : sequence_to_kmers(sequence, canonical_mode_)) {
+    for (const auto &[kmer, is_valid] : sequence_to_kmers(sequence, mode_ != BASIC)) {
```

Comment:
so DBGBitmap with the `PRIMARY` flag is essentially a graph which only stores the canonical k-mers? maybe we should mention this in the README somewhere

Reply:
Yeah, but I think it goes too far.
It's good if we merge the wrapper into the graphs, but right now it's not perfect, because one can't extract contigs from such a primary graph. I'll change this, so it will work as the succinct graph -- take all input sequences and make the user responsible for not passing a k-mer together with its rev-compl.

Reply:
done